platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/renderer/egl/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Contains the Mapbox Maps Android egl API classes.
```

Comment:
Looks like "EGL" is usually written in caps (e.g. https://www.khronos.org/egl), so we should probably do the same here?